build/ci-build.yml

```diff
@@ -12,7 +12,7 @@ pr:
   paths:
     include:
       - src/*
-      - build/ci-build.yml
+      - build/*
```

Comment:
Was first looking into adding the `README.md` file to the PS module, so that it can b shown in the Gallery, but saw that this was not how the gallery worked, so removed that one. This change, though, was made bc PR should be triggered if any change happens in the folder and not only the build script.